### PR TITLE
Update project to use eslint-config-braintree

### PR DESCRIPTION
**Pull request by crookedneighbor**
@braintree/front-end

### CODE REVIEW

File: test/form.js [hunk 1]
```diff
@@ -162,7 +180,7 @@ describe('Form', function () {
       this.form.submitHandler = 'fakeHandler';
       this.form.detach();
 
-      expect(this.form.submitHandler).to.equal(undefined);
+      expect(this.form.submitHandler).to.not.exist;
```
Comment (EvanHahn) [inline test/form.js]: This is slightly different behavior. Could we keep it as `.to.equal(undefined)`?

**Comment by EvanHahn** [comment]
:+1: other than my comment

File: test/form.js (hunk 1 above)
Comment (crookedneighbor) [inline test/form.js]: Sure. I'll just add the `eslint-disable-line` comment.

**Comment by mrak** [comment]
👍

**Pull request merged**